src/cli.rs

```diff
@@ -462,6 +462,10 @@ pub struct Misc {
     /// Defaults to the number of CPUs
     #[clap(env, long)]
     pub threads: Option<usize>,
+
+    /// How frequently to refresh statistics about API boundary nodes
```

Comment:
Why don't you put it under the metrics group around line 300?

Reply:
I thought that `Metrics` struct is intended for something more generic, like endpoint. But i can put it here, no problem.

Reply:
+1

Reply:
@blind-oracle `+1` means leave as is?

Reply:
Means move to metrics :)

Reply:
done